src/Textarea.jsx

```diff
@@ -382,39 +345,4 @@ class ReactTextareaAutocomplete extends React.Component {
   }
 }
 
-const triggerPropsCheck = ({ trigger }: { trigger: triggerType }) => {
```

Comment:
Please put the PropTypes back, not everyone uses the Flowcheck and the Proptypes's runtime checking is good, don't you think? 👍

Reply:
I read some posts and talked with the guys here on my work, so I guess the idea of flow is a replacement of `proptypes` then to avoid runtime errors, you know? 

I still put again the `proptypes` back to test, but the `eslint` screams at with me asking where are all `proptypes` of class. Then I think would be redundant if we put every `proptypes` again, what do you think? I could be wrong 😄

Reply:
You are right, Flowtype is a replacement for Proptypes, but for sure (from the nature of type checking) is not capable to avoid runtime errors. That is the domain for Proptypes, also not everyone will use our es6 code with Flowtypes support (because he will be using Create React App for example) so the support at least of Protypes would be handy, especially because Proptypes will be watching for `triggerProps` really in deep to avoid any misunderstanding.

Reply:
I know that there is a plugin for the automatic PropTypes export, but I think that it's halfway solution.

Reply:
I think there is no reason to create the `PropTypes` manually, and there is no problem on use `babel` to do it.

Reply:
@dleitee Could you please provide me an example of how to do it another way with preserving of current behavior? If there is any I would like to do it / approve it, but I'm not aware of any.